ehrapy/api/preprocessing/encoding/_encode.py

```diff
@@ -151,17 +157,24 @@ def _encode(
 
         # user passed categorical values with encoding mode for each of them
         else:
+            # reencode data
```

Comment:
Performance penalty? Can you do an ugly test with a very large matrix again, please?

Reply:
Will do but keep in mind that this is basically the same like one would encode an unencoded AnnData object with all encodings that were changede and or added.

Reply:
As discussed